deploy/Dockerfile_debug

```diff
@@ -46,6 +46,9 @@ COPY --from=0 /usr/local/bin/docker-credential-gcr /kaniko/docker-credential-gcr
 COPY --from=0 /go/src/github.com/awslabs/amazon-ecr-credential-helper/bin/linux-amd64/docker-credential-ecr-login /kaniko/docker-credential-ecr-login
 COPY --from=0 /usr/local/bin/docker-credential-acr-linux /kaniko/docker-credential-acr
 COPY --from=1 /distroless/bazel-bin/experimental/busybox/busybox/ /busybox/
```

Comment:
Based on https://github.com/GoogleContainerTools/kaniko/issues/1277 - distroless provided busybox does not add any extra. So the `# Workaround comment` can be removed.